pandas/tests/io/parser/test_na_values.py

```diff
@@ -111,10 +111,10 @@ def f(i, v):
         elif i > 0:
             buf = "".join([","] * i)
 
-        buf = "{0}{1}".format(buf, v)
+        buf = f"{buf}{v}"
 
         if i < nv - 1:
-            buf = "{0}{1}".format(buf, "".join([","] * (nv - i - 1)))
+            buf = f"{buf}{''.join([','] * (nv - i - 1))}"
```

Comment:
If you can assign this to a variable on the previous line should help readability

Reply:
Do you have a variable name suggestion?

Reply:
```joined``` should do the trick IMO